iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SwitchTerminalForVoltage.java

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network.util;
```

Comment:
Why do you need this utility class?

Reply:
Removed after not allowing to define voltageAngleLimit on switches.